core/src/com/po/fuck/view/Sprites/BasicSpriteInfo.java

```diff
@@ -0,0 +1,54 @@
+package com.po.fuck.view.Sprites;
+
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.utils.JsonValue;
+
+/*
+ * assetsData.json description
+ * all textures have:
+ * each .png stores some table of frames for the object
+ * path - path to texture
+ * frameWidth - the width of single frame
+ * frameHeight - the height of single frame
+ * width - the total width of this texture
+ * height - the total height of this texture
```

Comment:
Probably should be changed, according to the comment inside of the `Player.java`